webpack/assets/javascripts/react_app/components/Pagination/Pagination.js

```diff
@@ -22,6 +24,7 @@ const Pagination = props => {
     ...otherProps
   } = props;
 
+  const { perPage } = getContext(SETTINGS);
```

Comment:
Can we use a selector here: e.g. `selectSettings, selectPerPage` which will be imported from `/Context` ?

Reply:
It's not redux state, wouldn't it confuse developers? data extraction from the context  differs from reducers.

Reply:
I think it is pretty clear if we import them from the `context` folder,
but that's just my opinion

Reply:
IMO we should think about a new convention to replace the `index.js` file with something like:
```js
const ConnectedPagination = ({ }) => (
  <Pagination
    perPage={getURIperPage() || getContext(SETTINGS).perPage}
  />
);
```

But I don't think it is related to this PR